feature_layers/feature-layer-dictionary-renderer/src/main/java/com/esri/samples/feature_layer_dictionary_renderer/FeatureLayerDictionaryRendererSample.java

```diff
@@ -54,7 +55,12 @@ public void start(Stage stage) {
     stage.setScene(scene);
     stage.show();
 
-    ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+    // create a map with a basemap style and set it to the map view
```

Comment:
there's a few more like this in this PR: I left a comment in an earlier PR to consider updating them, I'll leave it up to you if you have bandwidth to do so this time round and won't tag again in this PR. 

```suggestion
    // create a map with the topographic basemap style and set it to the map view
```